Biology.java

```diff
@@ -0,0 +1,346 @@
+                put("GTA", "V");
+                put("GTG", "V");
+
+
+                put("TCT", "S");
+                put("TCC", "S");
+                put("TCA", "S");
+                put("TCG", "S");
+
+                put("CCT", "P");
+                put("CCC", "P");
+                put("CCA", "P");
+                put("CCG", "P");
+
+                put("ACT", "T");
+                put("ACC", "T");
+                put("ACA", "T");
+                put("ACG", "T");
+
+                put("GCT", "A");
+                put("GCC", "A");
+                put("GCA", "A");
+                put("GCG", "A");
+
+
+                put("TAT", "Y");
+                put("TAC", "Y");
+                put("TAA", "STOP");
+                put("TAG", "STOP");
+
+                put("CAT", "H");
+                put("CAC", "H");
+                put("CAA", "Q");
+                put("CAG", "Q");
+
+                put("AAT", "N");
+                put("AAC", "N");
+                put("AAA", "K");
+                put("AAG", "K");
+
+                put("GAT", "D");
+                put("GAC", "D");
+                put("GAA", "E");
+                put("GAG", "E");
+
+
+                put("TGT", "C");
+                put("TGC", "C");
+                put("TGA", "STOP");
+                put("TGG", "W");
+
+                put("CGT", "R");
+                put("CGC", "R");
+                put("CGA", "R");
+                put("CGG", "R");
+
+                put("AGT", "S");
+                put("AGC", "S");
+                put("AGA", "T");
+                put("AGG", "T");
+
+                put("GGT", "G");
+                put("GGC", "G");
+                put("GGA", "G");
+                put("GGG", "G");
+            }};
+        }
+
+        /**
+         * Translates and returns the amino acid specified by the given codon
+         *
+         * @param codon the sequence of three nucleotides to translate
+         * @return the corresponding amino acid
+         */
+        public String getAminoAcid(String codon) {
+            return this.codonMap.get(codon);
+        }
+    }
+
+    /**
+     * Matrices of vectors to determine where to move virus in Euclidean space
+     *
+     * Each site is drawn from the gamma distribution
+     */
+    public enum SiteMutationVectors  {
+        VECTORS();
+
+        private final int totalSites = Parameters.startingSequence.length() / 3;
+
+        public final Map<Integer, double[][][]> siteMutationVectors;
+        public final HashSet<Integer> epitopeSites;
+        public final String[] stringOutputCSV = new String[totalSites]; // String[] to create CSV from
+
+        SiteMutationVectors() {
+            HashSet<Integer> epitopeSitesSet = new HashSet<>();
+            for (int i = 0; i < Parameters.epitopeSites.length; i++) {
+                epitopeSitesSet.add(Parameters.epitopeSites[i] - 1); // Allow users to index starting at 1, but store values starting at 0
```

Comment:
Could we add something to `description.md` that says that epitope sites should be indexed at 1.0?